### ISSUE TITLE
Fix phone locked alts

### ISSUE THREAD

**Issue by whendiscordsus** [Vedza/NitroSniperGo#316]
**Describe the bug**
In previous versions of NSG, alts that were phone locked could still snipe codes because the Discord gateway still allows phone locked accounts to connect and receive new message events. However, the latest update has broken this (maybe accidental?) functionality by trying to hit `/gateway/bot` before starting up. When the phone locked alt tries to hit the endpoint, it's blocked because phone locked accounts cannot use any endpoints until they're unlocked again.

**To Reproduce**
1. Add a phone locked token to your list of alts
2. Try to start up NSG
3. Observe error message (screenshot 1)
4. NSG quits

**Expected behavior**
Phone locked accounts should still be able to work because they're only blocked from using the API and not the gateway.

**Screenshots**
![Error message shown in terminal on startup](https://user-images.githubusercontent.com/85244453/120511313-eb477300-c38f-11eb-9221-7adae02c4ff6.png)

**Additional context**
Downgrading to any version of NSG before the custom library was implemented will resolve the issue.

**Comment by Vedza** [comment]
The previous version could not see content of messages after a discord API update. So maybe that discord update was made to prevent that exploit.

**Comment by whendiscordsus** [comment]
I never had issues using the previous versions with my phone locked alt, as afaik the gateway is still fully (or mostly) functional while an account is phone locked. Theoretically, when connected to the gateway you should always be able to receive `MESSAGE_CREATE` events with the message content but you shouldn't be able to hit the API endpoint to fetch the message again.

**Comment by Vedza** [comment]
Went back to the old version where everything was working fine. Please open a new issue if this issue still exists
